install.sh

```diff
@@ -9,24 +9,33 @@ if ! command -v unzip >/dev/null; then
 	exit 1
 fi
 
+function version { echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'; }
+
+if [ $# -gt 0 ]; then
+    deno_version=$1
+fi
+
 if [ "$OS" = "Windows_NT" ]; then
 	target="x86_64-pc-windows-msvc"
 else
 	case $(uname -sm) in
 	"Darwin x86_64") target="x86_64-apple-darwin" ;;
 	"Darwin arm64") target="aarch64-apple-darwin" ;;
 	"Linux aarch64")
```

Comment:
I don't think we need to worry about a version test here -- we can just enable support.

Reply:
Sounds good to me 🤝